com.opentabletdriver.OpenTabletDriver.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: com.opentabletdriver.OpenTabletDriver
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: start-otd.sh
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --share=network
+  - --device=all
+  - --filesystem=home
+modules:
+  - name: libnotify
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/libnotify.git
+        commit: 0.8.3
+    build-commands:
+      - meson _build --prefix=/app -Dtests=false -Dintrospection=enabled -Dgtk_doc=false -Ddocbook_docs=disabled -Dman=false
+      - meson compile -C _build
+      - meson install -C _build
+
+
+  - name: libevdev
+    sources:
+      - type: archive
+        url: "https://www.freedesktop.org/software/libevdev/libevdev-1.13.1.tar.xz"
+        sha256: "06a77bf2ac5c993305882bc1641017f5bec1592d6d1b64787bad492ab34f2f36"
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - ./configure --prefix=/app/
+      - make 
+      - make install
+
+  - name: opentabletdriver
+    buildsystem: simple
+    build-options:
+      append-path: "/usr/lib/sdk/dotnet6/bin"
+      append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+      build-args:
+        - --share=network
+      env:
+        PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
```

Comment:
use `append-pkg-config-path` in `build-options` instead.

Reply:
I'm a newbie. Thanks for the review. I will try to correct it.